### PR TITLE
Prototype2

### PR DESCRIPTION
**Pull request by tantam1992**
First commit prototype2

### CODE REVIEW

File: sensors/depth_cam.urdf.xacro [hunk 1]
```diff
@@ -80,7 +80,7 @@
         </camera>
         <plugin name="${name}_controller" filename="libgazebo_ros_openni_kinect.so">
           <baseline>0.2</baseline>
-          <alwaysOn>true</alwaysOn>
+          <alwaysOn>true</alwaysOn>roslaunch navigation navigation.launch map:=aws_hospital planner:=dwa
```
Comment (JosefGst) [inline sensors/depth_cam.urdf.xacro]: I guess the roslaunch command got accidantely copied in here. please remove.

File: sensors/depth_cam_LR.urdf.xacro [hunk 2]
```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0"?>
```
Comment (JosefGst) [inline sensors/depth_cam_LR.urdf.xacro]: I see you created copies of the depth cam xacro. Can we have only one xacro and use that single xacro for all 5 cameras?

**Review by JosefGst — CHANGES_REQUESTED** [review]

File: sensors/depth_cam_LR.urdf.xacro (hunk 2 above)
Comment (tantam1992) [reply sensors/depth_cam_LR.urdf.xacro]: yes, I will delete it

File: sensors/depth_cam.urdf.xacro (hunk 1 above)
Comment (tantam1992) [reply sensors/depth_cam.urdf.xacro]: OK

File: sensors/depth_cam_LR.urdf.xacro (hunk 2 above)
Comment (JosefGst) [reply sensors/depth_cam_LR.urdf.xacro]: Thank you. If the simulation still works I merge :smile:

**Pull request merged**